packages/dev/core/src/Engines/Extensions/engine.views.ts

```diff
@@ -156,6 +161,67 @@ Engine.prototype.unRegisterView = function (canvas: HTMLCanvasElement): Engine {
     return this;
 };
 
+Engine.prototype._renderViewStep = function (view: EngineView, parent: HTMLCanvasElement): boolean {

```

Comment:
Why is the "parent" needed? this is private/internavl, only used here, andboth parents passed here come from `this.getRenderingCanvas()` . Why not simply use `getRenderingCanvas`?